requirements.txt

```diff
@@ -19,4 +19,10 @@ albumentations>=1.3.0
 pytube>=12.1.3
 pyffmpeg==2.0
 GitPython>=3.1.18
+pytorch-metric-learning==1.6.3
+faiss-cpu==1.7.4
+hawk-eyes==2.1.2
+onnx==1.14.1
```

Comment:
Is onnx 1.15.0 incompatible?

Reply:
We can use 1.15.0, I will update the requirement.

Reply:
Updated in the latest commit.